packages/shared/src/components/modals/common.tsx

```diff
@@ -196,6 +200,7 @@ export const modals = {
   [LazyModal.ReportSource]: ReportSourceModal,
   [LazyModal.UserFollowersModal]: UserFollowersModal,
   [LazyModal.UserFollowingModal]: UserFollowingModal,
+  [LazyModal.Report]: ReportModal,
```

Comment:
Why is this called report? think it confuses things, I believe it's best to be called PostRejectModal